java-manual/modules/ROOT/pages/reactive.adoc

```diff
@@ -169,16 +176,17 @@ public class App {
 }
 ----
 
+
+[#session-creation]
 == Always defer session creation
 
-It's important to remember that in reactive programming *a Publisher doesn't come to life until a Subscriber attaches to it*.
-A Publisher is just an abstract description of your asynchronous process, but it's only the act of subscribing that triggers the flow of data in the whole chain.
+In reactive programming, *a Publisher doesn't come to life until a Subscriber attaches to it*: a Publisher is just an abstract description of your asynchronous process, but it's only the act of subscribing that triggers the flow of data in the whole chain.
```

Comment:
Technically, it's not fully true. There are also connectable publishers,  that won't even publish until connected or a publisher *may* prefetch. But I don't have a better way of phrasing it, really.